### PR TITLE
Fix slider behavior in IE 11

### LINKED ISSUE #22: License statement seems off
The license statement in the README.md file reads:

EventDrops is released under the MIT License, courtesy of marmelab and Canal Plus.

Unsure what EventDrops has to do with this repo.

### PR DESCRIPTION
**Pull request by cshinn**
Updates slider behavior and styling so that it functions correctly in IE 11. Also updates README and alphabetizes css file.

Closes #22

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -14,32 +14,32 @@
...
-  text-rendering: auto;
-  font-size: 10px;
+  cursor: pointer;
   font-family: 'FontAwesome';
   opacity: .7;
   stroke: none;
-  cursor: pointer;
+  font-size: 10px;
+  text-rendering: auto;
 }
```
Comment (jeff-phillips-18) [inline style.css]: nit: looks like you are keeping alphabetical order everywhere else but not here

File: style.css [hunk 2]
```diff
@@ -1,11 +1,11 @@
 .timeline-pf-zoom-area {
-  fill: transparent;
   cursor: pointer;
+  fill: transparent;
 }
 
 .timeline-pf-y-tick {
-  stroke: black;
   fill: none;
+  stroke: black;
```
Comment (jeff-phillips-18) [inline style.css:8]: Would be nice to use less and include patternfly variables for colors. Not in the scope of this PR though.

**Review by jeff-phillips-18 — APPROVED** [review]
👍

File: style.css (hunk 1 above)
Comment (cshinn) [reply style.css]: bah!

**Review by dgutride — APPROVED** [review]

**Pull request merged**